src/components/util/TermSelect.vue

```diff
@@ -119,4 +156,15 @@ const toggleTermLocked = () => {
 .select-term {
   max-width: 200px;
 }
+:deep(.v-switch .v-selection-control:not(.v-selection-control--dirty) .v-selection-control__input .v-icon) {
+  color: green;
```

Comment:
@mohamalnadi all looks great! Except I would use `rgb(var(--v-theme-success))` instead of `green`, and similarly substitute `error` for `red`. That way we know it works in both light and dark mode with sufficient color contrast.